vignettes/design_principle.Rmd

```diff
@@ -278,7 +278,9 @@ The modules and surrogate functions will depend mainly on the following packages
   [{matchmaker}](https://CRAN.R-project.org/package=matchmaker) utilized to perform the dictionary-based cleaning,
   [{lubridate}](https://CRAN.R-project.org/package=dplyr) used to create, handle, and manipulate objects of type Date,
   [{reactable}](https://CRAN.R-project.org/package=reactable) mainly used here to customize the data cleaning report,
-  [{withr}](https://CRAN.R-project.org/package=withr) utilized to handle the creation of temporary files and directory relevant for `print_report()` and,
+  [{cli}](https://CRAN.R-project.org/package=cli) utilized to reformat all the messages in the package,
+  [{potools}](https://CRAN.R-project.org/package=potools) specifically used to translate all the messages in the package into other languages such as French, Spanish, etc,
+  [{rlang}](https://CRAN.R-project.org/package=rlang) mainly used here for its `.data` functionality to retrieves data-variables from the data frame,
```

Comment:
```suggestion
  [{rlang}](https://CRAN.R-project.org/package=rlang) mainly used here for its `.data` functionality to retrieve data-variables from the data frame,
```

Is "mainly" necessary here? For which other purposes is rlang used? If `.data` is the only one, please remove "mainly" for clarity.

Reply:
I have remove `mainly`.